news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,211 @@
+
+Having since departed the team and osu! in general to pursue things elsewhere, we'd like to express just how thankful we are for his assistance over the years.
+
+### Stefan
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Stefan.jpg)](https://osu.ppy.sh/users/626907)
+
+The very picture of an unsung hero, Stefan has single-handedly maintained the [beatmap packs](https://osu.ppy.sh/beatmaps/packs) by himself for years regularly and consistently, completely without prompting and entirely out of a drive to keep the project alive.
+
+Thousands of users have Stefan to thank for their regular bundle of fresh maps to play, and we're extremely thankful for his incredible dedication and persistence.
+
+### Naxess
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Naxess.jpg)](https://osu.ppy.sh/users/8129817)
+
+On the subject of unsung heroes, Naxess' has pioneered the development of a suite of tools that have essentially come to shape the way the Beatmap Nominators and the Quality Assurance Team handles the constant influx of new beatmaps and things to check.
+
+Once Aiess and now Victoria, Naxess' creation stands guard in many a Discord server to guide masses of modders as they navigate through the challenges that come with ranking beatmaps, providing a service that hundreds of people now take largely for granted.
+
+His efforts have not gone unnoticed.
+
+### Kurokami
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Kurokami.jpg)](https://osu.ppy.sh/users/260933)
+
+A long term member of the Beatmap Nominators, then the Quality Assurance Team, Kurokami has been a regular presence in beatmapping-related affairs for literal years. His most notable contribution is his leadership of the Beatmap Spotlights project, and its continued effort to bring only the highest quality of beatmaps to bear for the community's appraisal.
+
+Through thick and thin, Kurokami's ceaseless direction of the project has kept it alive where circumstance would have doomed it to be forgotten, and it is this sheer determination and drive to see something done that we are tremendously grateful to benefit from.
+
+### p3n
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/p3n.jpg)](https://osu.ppy.sh/users/123703)
+
+Our initial reaction to p3n's name being raised was universally: "how does he not have this badge already?"
+
+So vast and numerous are p3n's contributions over the years - from creating a major metric used by the Quality Assurance Team to weigh and compare the activity of a modder in a given period of time, to entire written drafts and improvements on the basic modding/ranking system, his previous and enduring leadership of both the Beatmap Approval Team and the current QAT, assistance with the osu! World Cups and so, so much more.
```

Comment:
this is a REALLY long sentence. Can we somehow split this list up with a full stop?

Reply:
it's intentionally phrased this way to highlight the immense breadth of his contributions